tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -891,6 +891,9 @@ public:
    RResultPtr<::TH1D> Histo1D(const TH1DModel &model = {"", "", 128u, 0., 0.}, std::string_view vName = "")
    {
       const auto userColumns = vName.empty() ? ColumnNames_t() : ColumnNames_t({std::string(vName)});
+
+      const auto validatedColumns = GetValidatedColumnNames(1, userColumns);
+
```

Comment:
`CreateAction` below does call `GetValidatedColumnNames`...why does this addition fix the crash? :thinking: